src/EditorFeatures/CSharpTest/SymbolKey/SymbolKeyCompilationsTests.cs

```diff
@@ -160,7 +160,6 @@ public partial class C1
         var implementation = definition.PartialImplementationPart;
 
         // Assert that both the definition and implementation resolve back to themselves
-        // https://github.com/dotnet/roslyn/issues/73772: add a similar test for properties
```

Comment:
Was the mentioned test added?

Reply:
Seems like it's already below: ExtendedPartialPropertyDefinitionAndImplementationResolveCorrectly